calendars/numpy.yaml

```diff
@@ -66,13 +47,13 @@ events:
       follow the link: https://hackmd.io/oB_boakvRqKR-_2jRV-Qjg
 
       Join us via Zoom: https://numfocus-org.zoom.us/j/85016474448?pwd=TWEvaWJ1SklyVEpwNXUrcHV1YmFJQT09
-    begin: 2022-11-21 16:00:00 +00:00
-    end: 2022-11-21 17:00:00 +00:00
+    begin: 2023-10-09 23:00:00 +00:00
+    end: 2023-10-09 24:00:00 +00:00
```

Comment:
I think this should be `2023-10-10 00:00:00 +00:00`

Reply:
You're absolutely right, @stefanv! I've corrected it.